package.json

```diff
@@ -11,7 +11,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov",
+    "test": "node node_modules/lab/bin/lab -a code -t 100 -L",
```

Comment:
Copied from the Makefile... if it works without take it out (and open a PR on hapi)